sql/mob_droplist.sql

```diff
@@ -15760,7 +15760,7 @@ INSERT INTO `mob_droplist` VALUES (2935,0,0,1000,853,55);
 INSERT INTO `mob_droplist` VALUES (2936,0,0,1000,19123,200);
 INSERT INTO `mob_droplist` VALUES (2937,0,0,1000,2853,26);
 INSERT INTO `mob_droplist` VALUES (2938,0,0,1000,2842,222);
-INSERT INTO `mob_droplist` VALUES (2939,0,0,1000,16296,268);
+INSERT INTO `mob_droplist` VALUES (2939,0,0,1000,16296,214);
```

Comment:
http://ffxidb.com/zones/107/tococo

I think this would fall into very common (240)